test_docopt.py

```diff
@@ -65,13 +65,11 @@ def test_commands():
 
 
 def test_formal_usage():
-    doc = """
-    Usage: prog [-hv] ARG
-           prog N M
-
-    prog is a program."""
+    doc = '\n'.join(('Usage: prog [-hv] ARG',
+                     '       prog N M',
+                     'prog is a program.'))
```

Comment:
The `join` looks like a more sane way to have properly indented multiline strings.